platform-tests/upstream/run_acceptance_tests.sh

```diff
@@ -7,6 +7,23 @@ if  [ "${DISABLE_CF_ACCEPTANCE_TESTS:-}" = "true" ]; then
   exit 0
 fi
 
+# FIXME: Remove this once we are deploying a version of cf-release
+# that includes capi-release >= 1.35.0.
+(
+  cd  "$(pwd)/cf-release/src/github.com/cloudfoundry/cf-acceptance-tests/"
```

Comment:
I think this is fine as is. It would be harder to add the check for the expected commit SHA using a resource, and would also require quite a bit more restructuring of how we pull these in (switching from using the submodule to a separate git resource).